strings/redpill.json

```
@@ -1,42 +1,42 @@
-		"Separation is absolute.",
-		"Space stations are no substitute for healthy social interaction.",
-		"Space wind? How does that even make sense?",
-		"The gods are always watching, and will wipe you from existence if you fuck up.",
-		"The ultimate god is really really stoned.",
-		"The universe always ends after we reach CentCom.",
-		"There's no biological difference between lizards and humans.",
-		"This is all an endless looping nightmare of misery.",
-		"What happened to the ceiling?",
-		"What the fuck is carbon dioxide?",
-		"Where does the Cook get all this meat from?",
-		"Who hires all these unqualified, violent assistants?",
-		"Why are there fire alarms everywhere but no sprinklers?",
-		"Why do people just randomly murder all their friends some shifts?",
-		"Why does Nanotrasen hire Clowns and Mimes for every single station?",
-		"Why does space circle around on itself?",
-		"Why is it called the emergency shuttle if it arrives every single shift?",
-		"Why is space cold?",
-		"Why is the station's air supply connected to the plasma tank?",
-		"Why is there a floor, but no roof?",
-		"Why is this a plasma research station if we know everything about plasma already?",
-		"Why is this station so poorly designed?"
+		"2010 год был 550 лет назад.",
+		"Весь экипаж — это просто грейтайдеры с разной окраской.",
+		"Всеми управляют ниточки за светящимся экраном.",
+		"Как спасательная капсула может взять только двух человек обычно, но бесконечное количество людей, если они лежат?",
+		"Почему дыра в полу не высасывает тебя в космос?",
+		"Как я узнаю имя человека, просто услышав его голос?",
+		"Откуда я знаю имена всех этих людей, которых никогда раньше не встречал?",
+		"Как зеркала делают людям стрижки?",
+		"Как лотки в морге знают, есть ли душа ещё в теле?",
+		"Как все стали экспертами по каждой угрозе во вселенной?",
+		"Если магия реальна, почему мы её не исследуем?",
+		"Если датчики так важны, почему они не включаются сразу в последнем режиме?",
+		"Если бы два шахтёра были в Лаваленде и один из них убил другого киркой, разве это не было бы пиздецом?",
+		"Если мы можем стирать одежду в раковине, зачем нам стиральные машины?",
+		"Как же мало нужно, чтобы люди начали убивать своих друзей.",
+		"Нанотрейзен просто клонирует нас каждую смену.",
+		"Разделение абсолютно.",
+		"Космические станции не заменяют здорового социального взаимодействия.",
+		"Космический ветер? Как это вообще возможно?",
+		"Боги всегда наблюдают, и сотрут тебя из бытия, если ты проебёшься.",
+		"Высший бог на самом деле очень сильно обдолбан.",
+		"Вселенная всегда схлопывается после того, как мы достигаем ЦК.",
+		"Нет биологической разницы между ящерицами и людьми.",
+		"Это всё бесконечный повторяющийся кошмар.",
+		"Что случилось с потолком?",
+		"Что за хуйня этот углекислый газ?",
+		"Откуда повар берёт всё это мясо?",
+		"Кто нанимает всех этих некомпетентных, агрессивных ассистентов?",
+		"Почему везде пожарные сигнализации, но нет спринклеров?",
```

Речь идет про разбрызгиватели воды
```suggestion
		"Почему везде пожарные сигнализации, но нет системы пожаротушения?",
```